iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ShuntCompensatorLinearModelImpl.java

```diff
@@ -59,6 +59,16 @@ public int getMaximumSectionCount() {
         return maximumSectionCount;
     }
 
+    @Override
+    public double getCurrentB(int currentSectionIndex) {
```

Comment:
In that case, the currectSectionIndex is not an index: if it's an index return bPerSection, if it's a number of _in service_ section, change the parameter name